feddlib/core/Mesh/MeshUnstructured_def.hpp

```diff
@@ -129,15 +129,17 @@ template <class SC, class LO, class GO, class NO>
 void MeshUnstructured<SC,LO,GO,NO>::buildP2ofP1MeshEdge( MeshUnstrPtr_Type meshP1 ){
     
     // If flags of line segments should be used over surface flags, this functions must be checked
-    
-    int rank = this->comm_->getRank();
+     int rank = this->comm_->getRank();
     this->rankRange_ = meshP1->rankRange_;
     bool verbose( this->comm_->getRank() == 0 );
     this->elementMap_ = meshP1->elementMap_;
+	this->edgeMap_ = meshP1->edgeMap_;
     this->dim_ = meshP1->getDimension();
     this->FEType_ = "P2";
     this->numElementsGlob_ = meshP1->numElementsGlob_;
+	//this->surfaceTriangleElements_ = meshP1->surfaceTriangleElements_; // for later
     
+	meshP1->assignEdgeFlags(); // Function that determines the flag for each edge. That way the P2 flags can easily be determined
```

Comment:
Is it documented somewhere how the flag is determined if the two nodes have different flags?

Reply:
Mentioned in 'determineFlagP2(...)'. I could elaborate in meshUnstr. documentation

Reply:
We don't have to do it in this PR. Could also be on our todo list ;-) I just want to gradually improve this.

Reply:
Added :)